src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type Props = {
+  todos:Todo[];
+};
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <section>
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo key={todo.id} todo={todo} />
+      ))}
+    </section>
+  </section>
```

Comment:
Why did you use 2 `section` elements ?

```suggestion
    <section className="TodoList">
      {todos.map(todo => (
        <TodoInfo key={todo.id} todo={todo} />
      ))}
    </section>
```